cdap-storage-ext-spanner/src/main/java/io/cdap/cdap/storage/spanner/SpannerStructuredTableAdmin.java

```diff
@@ -212,7 +187,7 @@ private StructuredTableSchema loadSchema(StructuredTableId tableId) throws Table
     Statement schemaStatement = Statement.newBuilder(
       "SELECT C.column_name, C.spanner_type, I.index_type, I.ordinal_position FROM information_schema.columns C " +
         "LEFT JOIN information_schema.index_columns I " +
-        "ON C.column_name = I.column_name AND C.table_name = I.table_name " +
+        "ON C.column_name = I.column_name AND C.table_name = I.table_name AND I.ordinal_position is not NULL " +
```

Comment:
@chtyim I found that missing this `not NULL` condition leads to a repeatable index bug while loading the schema. e.g, for a table
```
StructuredTableSpecification UPDATED_SIMPLE_TABLE_SPEC =
    new StructuredTableSpecification.Builder()
      .withId(SIMPLE_TABLE)
      .withFields(
        Fields.intType("i"),
        Fields.stringType("s"),
        Fields.longType("l"),
        Fields.floatType("updated_field1"),
        Fields.longType("updated_field2"))
      .withPrimaryKeys(KEY_FIELD.getName())
      .withIndexes(STR_FIELD.getName(), LONG_FIELD.getName())
      .build();
```

Before:
<img width="1449" alt="Screen Shot 2022-08-29 at 12 02 17" src="https://user-images.githubusercontent.com/20428112/187281309-a5c22f46-256b-4738-81bb-22489245e671.png">

After:
<img width="1436" alt="Screen Shot 2022-08-29 at 12 04 05" src="https://user-images.githubusercontent.com/20428112/187281402-d00d7e2f-2ba7-485c-beff-b2c848ac6640.png">